portfolio/pom.xml

```diff
@@ -16,7 +16,8 @@
 
     <!-- Project-specific properties -->
     <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
-    <googleCloudProjectId>YOUR_PROJECT_ID_HERE</googleCloudProjectId>
+    <googleCloudProjectId>forellana-sps-spring21
+</googleCloudProjectId>
```

Comment:
Nit (short for nitpick): Let's keep this on one line so `<googleCloudProjectId>forellana-sps-spring21</googleCloudProjectId>`